storybook/src/stories/Tables/molecules/LoadingTable.stories.tsx

```diff
@@ -0,0 +1,148 @@
+import React, {useState, useCallback, useEffect} from 'react';
+import styled from 'styled-components';
+import { object, boolean, text } from "@storybook/addon-knobs";
+
+import {TypeTable as LoadingTable } from 'scorer-ui-kit';
+import photo from '../../assets/placeholder.jpg';
+
+import { 
+  ITableColumnConfig,
+  ITypeTableData
+} from 'scorer-ui-kit/dist/Tables';
+
+export default {
+  title: 'Tables/molecules',
+  component: LoadingTable,
+  decorators: []
+};
+
+const Container = styled.div`
+  padding: 100px;
+`;
+
+
+const columnConfigSample : ITableColumnConfig[] = [
+  {
+    header: 'When',
+    sortable: false,
+    cellStyle: 'normalImportance',
+    showStatus: true,
```

Comment:
Isn't a `showStatus` type of data.

Reply:
showStatus is boolean :)
Is the dot to the left.

By the way I will remove the extra dots in the other columns, that were added by mistake 🙈 

<img width="347" alt="Screen Shot 2021-03-11 at 19 22 29" src="https://user-images.githubusercontent.com/10409078/110772602-2198b900-829f-11eb-8b77-9db2b1b0ab28.png">

Reply:
Ahhh, of course it is! Thanks 👍